packages/react/dropdown-menu/src/DropdownMenu.stories.tsx

```diff
@@ -155,6 +155,33 @@ export const Modality = () => {
   );
 };
 
+export const TextSelection = () => (
```

Comment:
Left this in so you can test PR but I'll probably remove it once we're happy it's all working. Not sure this story is needed?

Reply:
It's a bit of an edge really, appreciated it being here to test but I don't think we need to keep it around imo.